docs/CloudPermissions.md

```diff
@@ -0,0 +1,58 @@
+# Assign Required Cloud Permissions For Deploying Scalar DL
```

Comment:
The permissions are mainly or fully for k8s, right? 
mainly? or fully?

In any cases, why don't you say it's k8s related in the title and the first sentence?
Otherwise, it's very misleading.

Reply:
>The permissions are mainly or fully for k8s, right?

The permissions are for managing all the resources that are needed for Scalar DL deployment, specified that in the first sentence.

Reply:
The question is `what do you mean Scalar DL?`
When deploying Scalar DL in k8s, Scalar DL components like scalar-ledger and scalar-envoy are just pods, so I think they don't require any permissions if k8s is properly created, do they?
In other words, if we want to deploy nginx pods and a LoadBalancer in front, then what permissions are needed? (the same permissions are required?)
So, I think the current description is too blur and somewhat wrong.